spec/requests/api/requests_spec.rb

```diff
@@ -228,6 +228,28 @@
       expect(response.parsed_body).to include(expected)
       expect(response).to have_http_status(:ok)
     end
+
+    it "exposes workflow in the request resources" do
+      ems = FactoryGirl.create(:ems_vmware)
+      vm_template = FactoryGirl.create(:template_vmware, :name => "template1", :ext_management_system => ems)
+      request = FactoryGirl.create(:miq_provision_request,
+                                   :requester => @user,
+                                   :src_vm_id => vm_template.id,
+                                   :options   => {:owner_email => 'tester@example.com'})
+      FactoryGirl.create(:miq_dialog,
+                         :name        => "miq_provision_dialogs",
```

Comment:
It's not clear to me why this object's attributes being specifically this way are important for the behavior being exercised/verified